scripts/simulator/.gitignore

```diff
@@ -0,0 +1,6 @@
+
+*.config.json
+*.credentials.json
+
+*.config.json5
+*.credentials.json5
```

Comment:
This ignore file will allow users to have different configurations to run and/or different environments to run simulations to.

NOTE: Double extension naming (`*` . `purpose` . `format`) allows for finer grained segmentation of files to reference.